android/clear_environment.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+
+# Script to clear environment BOINC for Android
```

Comment:
Verify that this script run from the correct folder, otherwise fail

Reply:
Can you suggest how to do it?

Reply:
https://github.com/BOINC/boinc/blob/005ace950810d8c467c6c06a7dca66e368506c10/3rdParty/buildLinuxDependencies.sh#L44-L48

Reply:
Thanks! that help! :)